website/docs/GettingStarted.md

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
Is this a new requirement from the Docusaurus to have docs under `website/`? We used to fetch them from the `docs/` directory. It would be cool to stay this way, but if we can't, let's move all the `docs/` to `website/`, so that they're not duplicated.

Reply:
Yes, I have fixed it